### PR TITLE
add Circle/CI files

### PR DESCRIPTION
**Pull request by brianwchou**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    docker:
+      - image: circleci/node:8.16.0-jessie
+    steps:
+      - checkout
+      - run:
+          name: Install
+          command: npm install
+      - run:
+          name: Compile
+          command: npx truffle compile
```
Comment (ejwessel) [inline .circleci/config.yml]: yarn truffle compile

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+      - run:
+          name: Test
+          command: npx truffle test
```
Comment (ejwessel) [inline .circleci/config.yml]: yarn truffle test

File: .circleci/config.yml (hunk 1 above)
Comment (ejwessel) [reply .circleci/config.yml]: The `truffle test` automatically compile. you can probably remove this section.

File: package-lock.json [hunk 3]
```diff
@@ -7,6 +7,786 @@
       "resolved": "https://registry.npmjs.org/assertion-error/-/assertion-error-1.1.0.tgz",
```
Comment (ejwessel) [inline package-lock.json]: I thought we were using yarn. If not let's use yarn.

File: package.json [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+  },
+  "dependencies": {},
+  "devDependencies": {
+    "truffle": "^5.0.5",
+    "ganache-cli": "^6.3.0",
+    "ganache-time-traveler": "^1.0.2",
+    "truffle-assertions": "^0.9.1"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```
Comment (ejwessel) [inline package.json]: add the compile and test commands in here. feel free to reference Hangman
https://github.com/brianwchou/Hangman/blob/master/package.json

**Review by ejwessel — CHANGES_REQUESTED** [review]
Some minor changes.

File: package.json (hunk 4 above)
Comment (brianwchou) [reply package.json]: I don't think we need them. I just ran yarn on the build file and it worked on its own.

Comment (ejwessel) [reply package.json]: an yes, but i'd like to `yarn compile` in addition to `yarn test`

**Review by ejwessel — APPROVED** [review]
LGTM

**Pull request merged**